src/tx/index.js

```diff
@@ -0,0 +1,71 @@
+const sig = require("@tendermint/sig");
+const _ = require("lodash");
+const axios = require("axios");
+
+const api = {
+  getAccount: "/auth/accounts",
+  postTx: "/txs"
+};
+
+async function getTx(path, base) {
+  return await axios.get(new URL(path, base).toString());
```

Comment:
on node `v8.10.0` I get an error here saying URL is not defined. Adding
```
const URL = require('url').URL
```
to imports fixed it

Reply:
https://github.com/Kava-Labs/javascript-sdk/pull/1/commits/3983081920e166eea012ca5ee1a0700598ee2bf2